src/rendering/custom-element-widget.js

```diff
@@ -107,18 +107,24 @@ function makeInit(tagName, definitionFn) {
   return function initCustomElement() {
     //console.log('%cInit() custom element ' + tagName, 'color: #880088');
     let widget = this;
+    widget.disposable = new Rx.CompositeDisposable();
+
     let element = createContainerElement(tagName, widget.properties);
     let propertiesProxy = makePropertiesProxy();
     let domUI = applyToDOM(element, definitionFn, propertiesProxy);
     element.cycleCustomElementMetadata = {
       propertiesProxy,
       rootElem$: domUI.rootElem$,
-      customEvents: domUI.customEvents,
-      eventDispatchingSubscription: false
+      customEvents: domUI.customEvents
     };
-    element.eventDispatchingSubscription = subscribeDispatchers(element);
```

Comment:
Why remove subscribeDispatchers? If I remember correctly, the synthetic-examples/nested-custom depends on it to work correctly.